Add compel section (#76)
MiniFate/MiniFate: _chapters/05-aspects-and-fate-points.md

{% comment %} Make sure to mention that players should be on board and have
fun with compels. If they aren't, then it's wrong.{% endcomment %}

Sometimes the GM will come up with a fun twist that makes a PC's life
difficult. In this case they may compel a PC: the GM explain how a certain
aspect would lead to the story unfolding in a manner unfavorable to the PC and
the player may decide to accept the narration in exchange for a fate point.

The exact way in which things go wrong can be negotiated. If the player
accepts the compel, they receive a Fate Point and the GM narrates the results.
There are no rolls made and no skills are used, the story simply unfolds as
the GM dictates.

The GM can compel a character aspects:

> Tor Byorn and Naka have snuck into Lord Kellett's chambers looking for a
> missing artifact and have stopped to discus what to do with the large safe
> they found. While they talk, the GM turns to Naka's player, holds up a Fate
> Point, and says, "Naka, I am pretty sure you have ~~Never met a problem your

> > magic couldn't ignite~~! Just melt the safe. Of course, the whole room will
> > be engulfed in flames alerting every guard in the building to your
> > whereabouts but you can handle them."

Reviewer: ```suggestion
> magic couldn't ignite~~! Just melt the safe. What's the worst that could happen?"
```

{% comment %} Make sure to mention that players should be on board and have
fun with compels. If they aren't, then it's wrong.{% endcomment %}

Sometimes the GM will come up with a fun twist that makes a PC's life
difficult. In this case they may compel a PC: the GM explain how a certain
aspect would lead to the story unfolding in a manner unfavorable to the PC and
the player may decide to accept the narration in exchange for a fate point.

The exact way in which things go wrong can be negotiated. If the player
accepts the compel, they receive a Fate Point and the GM narrates the results.
There are no rolls made and no skills are used, the story simply unfolds as
the GM dictates.

> Compels add excitement and unpredictability to the story at the cost of a

Author: Drafted this section in a frenzy as my kids charged up the stairs. Needs work!
